docs/development/index.md

```diff
@@ -0,0 +1 @@
+# Developing this Package
```

Comment:
We can add HTML meta stuff once there is content from which we can determine the HTML meta stuff.

```suggestion
# Developing this package
```